department-of-veterans-affairs/caseflow
# Do not show decided issues in the reader sidebar

Resolves #8761

### Description
This removes issues that have already been decided from Claims folder details and Issues sidebar in Document Viewer

### Acceptance Criteria
- [X] Issues with dispositions A through X are filtered from the Claims folder details
- [X] Issues with dispositions A through X are filtered from the Issues sidebar in Document Viewer

### Testing Plan
1. Go to /queue as BVAAABSHIRE (a judge, but they have an appeal with decided issues)
2. Go to detail view for Grover E. Kerluke (/queue/appeals/3449656), confirm the last 3 issues have disposition A - Advance Allowed in Field
3. Go to the Claims folder (reader/appeal/3449656/documents), expand Claims folder details, and confirm the last 3 issues are omitted.
4. View any document from the case and confirm that the last 3 issues do not appear in the Document Information Issues section
